### DISCUSSION TITLE
Credenziali

### DISCUSSION THREAD

**Discussion by AlexRux** [aturri/ha-zcsazzurro#33] [Q&A] [answered]
Ma che credenziali devo usare? Non capisco e non è descritto cosa serve inserire.

**Comment by aturri** [comment, marked as answer]
Ciao, devi chiedere le credenziali per accesso alle API al supporto ZCS. Trovi tutto nel readme: https://github.com/aturri/ha-zcsazzurro

**Reply by morettoni** [reply]
Hello I asked the credential to ZCS with the "support" form in their website but I didn't received any reply from them... is there any other way to get my API access to my data?
Thanks in advance for the help!

**Comment by weisen-p** [comment]
I also have problems to find the right way to get the credentials for my inverter. I have filled out the form here:
https://www.zcsazzurro.com/ > SUPPORT > REQUEST ASSISTANCE AND SALES INFO > Post Sales > ...
Is that the correct way to get the auth_key and the client_code?
If not, can anyone describe the correct way please?

**Reply by grzwolf** [reply]
Same here, didn't get any answer from zcsazzurro,com support. 
The Austrian Azzurro support support@zcsazzurro.at made the difference, they answered right away and provided the requested credentials.

**Comment by adesrumaux** [comment]
Hello, 

I got my credentials from ZCS but I'm not sure if I put them correctly in my configuration file (sorry I'm new in HA). How Can I check the logs?
I cannot see the module in my interface

Is this format correct in my configuration file?
zcsazzurro:
  auth_key: Zcs eKcsfderJd20
  client_code: XxxxxXxxxxxxx

Thank you in advance
Alex

**Reply by grzwolf** [reply]
Your /homeassistant/configuration.yaml snippet misses two spaces indentations (I never tried w/o).

```
zcsazzurro:
  auth_key: Zcs <your_key>
  client_code: <your_client_code>
```
  
Did you otherwise follow the installation / setup instructions provided 
at https://github.com/aturri/ha-zcsazzurro by word?

A ZSC debug logfile needs manual enabling _after_ a successful integration.

**Reply by adesrumaux** [reply]
Solved! I was missing this part:
**"After modifying this, restart Home Assistant and go to Integrations > Add Integration and select ZCS Azzurro. Sometimes you must refresh the browser cache to find the integration.

Pick serial number of your inverter / energy meter and insert it to complete the config flow: a new device with serial number inserted will appear. Add a new ZCS Azzurro config entry for each device you want to add."**
Thank you for your support
